lib/Gocdb_Services/User.php

```diff
@@ -540,6 +540,8 @@ public function getAuthTypes($reducedRealms=true) {
             if (strpos($authTokenName, 'Shib') !== false) {
```

Comment:
Not from changes but it seems a bit lazy to go to the trouble of creating unique, name-spaced authentication class identifiers and then just check the short substrings _Shib_, _X509_ and _IAM_.

Reply:
Added to #360